package-lock.json

```diff
@@ -19,7 +19,7 @@
     "abbrev": {
       "version": "1.1.1",
       "resolved": "https://registry.npmjs.org/abbrev/-/abbrev-1.1.1.tgz",
-      "integrity": "sha512-nne9/IiQ/hzIhY6pdDnbBtz7DjPTKrY00P/zvPSm5pOFkl6xuGrGnXn/VtTNNfNtAfZ9/1RtehkszU9qcTii0Q==",
+      "integrity": "sha1-+PLIh60Qv2f2NPAFtph/7TF5qsg=",
       "dev": true
```

Comment:
Hi @OmnesPotens thanks for making the changes! It seems that npm is converting the integrity hash from 512 to 1 for all the packages on your system. Please try to resolve this.

Here are a couple of link regarding the issue:

https://github.com/npm/npm/issues/16938
https://github.com/npm/npm/issues/17749

Maybe this will help:

'Ran into the same issue, fixed it by clearing the cache npm cache clean --force

manually deleting node_modules and then running npm install again.'

Reply:
@jkrooskos um, it looks like the integrity hash for that particular entry was original 1 and -not- 512. I just checked the main develop branch to make sure. I went through the troubleshooting from the links you posted just to make sure and cleaned my npm cache, but it didnt "fix" anything so I'm going to say its as it should be.

Reply:
Updating to the latest npmc experimental build also didnt help to solve the issue after using the aforementioned steps.

Reply:
hmm, seems to be a known issue. Just curious, what OS are you working with?

Reply:
I'm running Windows 10 on 2 different machines.